Userland/Utilities/man.cpp

```diff
@@ -31,7 +31,7 @@ static ErrorOr<pid_t> pipe_to_pager(DeprecatedString const& command)
     posix_spawn_file_actions_init(&action);
     posix_spawn_file_actions_adddup2(&action, stdout_pipe[0], STDIN_FILENO);
 
-    pid_t pid = TRY(Core::System::posix_spawnp("sh"sv, &action, nullptr, const_cast<char**>(argv), environ));
+    pid_t pid = TRY(Core::System::posix_spawnp("/bin/Shell"sv, &action, nullptr, const_cast<char**>(argv), environ));
```

Comment:
So why did you leave `argv[0]` the same?

Reply:
I assumed it's intended to run in POSIX compatibility mode.

Reply:
I have simplified it to `/bin/Shell -c command`